spec/time_date_spec.rb

```diff
@@ -0,0 +1,45 @@
+describe LinuxAdmin::TimeDate do
+  RUN_COMMAND = described_class.cmd("timedatectl")
+
+  def timedatectl_result
+    output = File.read(Pathname.new(data_file_path("time_date/timedatectl_output")))
+    AwesomeSpawn::CommandResult.new("", output, "", 0)
+  end
+
+  describe ".timezone" do
+    it "returns the correct timezone" do
+      awesome_spawn_args = [
+        RUN_COMMAND,
+        :params => ["status"]
+      ]
+      expect(AwesomeSpawn).to receive(:run).with(*awesome_spawn_args).and_return(timedatectl_result)
+      tz = described_class.timezone
+      expect(tz).to eq("America/New_York (EDT, -0400)")
```

Comment:
I feel like this method should match what you would pass into `timezone=`, though I can understand needing the details.  Perhaps a second method `timezone_detailed` or something?